components/explore/filters/StatusFilter.vue

```diff
@@ -26,8 +26,9 @@
 <script lang="ts" setup>
 import useReplaceUrl from './useReplaceUrl'
 import { NeoCheckbox } from '@kodadot1/brick'
+import { useExploreFiltersStore } from '@/stores/exploreFilters'
 
-const { $store } = useNuxtApp()
+const exploreFiltersStore = useExploreFiltersStore()
```

Comment:
```suggestion
const { listed, setListed, owned, setOwned } = useExploreFiltersStore()
```
I wonder if it could work as well.

Reply:
ts error `Cannot redeclare block-scoped variable 'listed'`, same for owned

Reply:
oh nvm it's because we declare `const listed = ...` in this file

Reply:
I'm not sure it's a good pratice to abstract main store (from multiple exemple I can see), I'd say keep it that way for now.

Otherwise people tend to do something like this for actions:
```js
const { setListed, setOwned } = exploreFiltersStore
```

Reply:
Oh, I found a demo here https://pinia.vuejs.org/core-concepts/. That seems a best practice.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/31397967/222168013-7fa9f22d-9c12-459d-b0fe-895484fc3749.png">

Reply:
We could keep the code that way for now.